config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
```

Comment:
we are taking a different approach and stepwise adopting these defaults or not. using separate commits. if you are loading defaults anyway then you should not need this and can get rid of i would have thought

Reply:
@jsugarman thanks, I have removed this file.